src/native/libs/CMakeLists.txt

```diff
@@ -144,6 +148,7 @@ if (CLR_CMAKE_TARGET_UNIX OR CLR_CMAKE_TARGET_BROWSER)
         add_compile_options(-Wno-typedef-redefinition)
         add_compile_options(-Wno-c11-extensions)
         add_compile_options(-Wno-thread-safety-analysis)
+        add_compile_options(-Wno-strict-prototypes)
```

Comment:
Does it need a `TODO-LLVM` to fix the warnings?

Reply:
Added a comment, and `-Wno-unused-but-set-variable` which was also failing in some pal files.